cluster/gce/gci/configure-helper.sh

```diff
@@ -480,7 +480,7 @@ function start-kubelet {
     if [[ ! -z "${KUBELET_APISERVER:-}" && ! -z "${KUBELET_CERT:-}" && ! -z "${KUBELET_KEY:-}" ]]; then
       flags+=" --api-servers=https://${KUBELET_APISERVER}"
       flags+=" --register-schedulable=false"
-      flags+=" --pod-cidr=10.123.45.0/30"
+      flags+=" --pod-cidr=10.123.45.0/29"
       reconcile_cidr="false"
     else
       flags+=" --pod-cidr=${MASTER_IP_RANGE}"
```

Comment:
what is master_ip_range? I'm guessing we can still hit the bug if it's set to something small, so depending on what/who uses it we should include a line in the https://github.com/kubernetes/kubernetes/issues/32332